packages/gatsby-image/gatsby-browser.js

```diff
@@ -0,0 +1,19 @@
+"use strict"
+
+exports.onClientEntry = () => {
+  if (process.env.GATSBY_IMAGE_LOAD_POLYFILLS) {
+    // IntersectionObserver polyfill for gatsby-image (Safari, IE)
+    if (typeof window.IntersectionObserver === `undefined`) {
```

Comment:
I think we should remove this in favor of documenting _how_ to polyfill app-level features like this.

Reply:
Sure! I get what you mean: IntersectionObserver has a number of use cases besides `gatsby-image`, so better to make an app-wide decision about whether to support it. 

Nothing breaks if Safari doesn't blur up nicely.